miniupnpd/Changelog.txt

```diff
@@ -810,7 +810,7 @@ VERSION 1.6 : released the 2011/07/25
   preparing SSDP stuff for IP v6. Trying to conform to UDA v1.1
 
 2011/03/09:
-  Some modifications thanks to Daniel Dickinson to improve OpenWRT
+  Some modifications thanks to Daniel Dickinson to improve OpenWrt
```

Comment:
I'm not a big fan of rewriting the Changelog.txt afterwards, but well, OK :)

Reply:
Ah, I did git sed, did not notice it touched Changelog.txt. If you want I'll revert the change for Changelog.txt?

Reply:
nevermind